[2023-07-13 06:04:01,063] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_user 2023-06-19T17:01:37+00:00 [queued]>
[2023-07-13 06:04:01,170] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_first_dag.create_user 2023-06-19T17:01:37+00:00 [queued]>
[2023-07-13 06:04:01,171] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 06:04:01,171] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2023-07-13 06:04:01,172] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-07-13 06:04:01,222] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): create_user> on 2023-06-19T17:01:37+00:00
[2023-07-13 06:04:01,226] {standard_task_runner.py:52} INFO - Started process 86519 to run task
[2023-07-13 06:04:01,238] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_first_dag', 'create_user', '2023-06-19T17:01:37+00:00', '--job-id', '9794', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag/test.py', '--cfg-path', '/tmp/tmpvycy8o45', '--error-file', '/tmp/tmp862mtt9c']
[2023-07-13 06:04:01,239] {standard_task_runner.py:77} INFO - Job 9794: Subtask create_user
[2023-07-13 06:04:01,332] {logging_mixin.py:104} INFO - Running <TaskInstance: my_first_dag.create_user 2023-06-19T17:01:37+00:00 [running]> on host d9c3ec86c3ab
[2023-07-13 06:04:01,656] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_first_dag
AIRFLOW_CTX_TASK_ID=create_user
AIRFLOW_CTX_EXECUTION_DATE=2023-06-19T17:01:37+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-19T17:01:37+00:00
[2023-07-13 06:04:01,659] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2023-07-13 06:04:01,660] {subprocess.py:63} INFO - Running command: ['bash', '-c', '\nid -u test_user;\nif [$? -eq o];\n    then echo "User exists"\nelse sudo useradd -r test_user;\n    echo "User created";\nfi']
[2023-07-13 06:04:01,673] {subprocess.py:75} INFO - Output:
[2023-07-13 06:04:01,742] {subprocess.py:79} INFO - id: ‘test_user’: no such user
[2023-07-13 06:04:01,749] {subprocess.py:79} INFO - bash: line 2: [1: command not found
[2023-07-13 06:04:01,814] {subprocess.py:79} INFO - 
[2023-07-13 06:04:01,815] {subprocess.py:79} INFO - We trust you have received the usual lecture from the local System
[2023-07-13 06:04:01,816] {subprocess.py:79} INFO - Administrator. It usually boils down to these three things:
[2023-07-13 06:04:01,816] {subprocess.py:79} INFO - 
[2023-07-13 06:04:01,817] {subprocess.py:79} INFO -     #1) Respect the privacy of others.
[2023-07-13 06:04:01,817] {subprocess.py:79} INFO -     #2) Think before you type.
[2023-07-13 06:04:01,818] {subprocess.py:79} INFO -     #3) With great power comes great responsibility.
[2023-07-13 06:04:01,819] {subprocess.py:79} INFO - 
[2023-07-13 06:04:01,819] {subprocess.py:79} INFO - sudo: no tty present and no askpass program specified
[2023-07-13 06:04:01,827] {subprocess.py:79} INFO - User created
[2023-07-13 06:04:01,831] {subprocess.py:83} INFO - Command exited with return code 0
[2023-07-13 06:04:01,897] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=create_user, execution_date=20230619T170137, start_date=20230713T060401, end_date=20230713T060401
[2023-07-13 06:04:01,969] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-13 06:04:02,012] {local_task_job.py:151} INFO - Task exited with return code 0
